src/model/attendee.py

```diff
@@ -32,51 +34,53 @@ def __init__(self, id, course, year, gender, status, email, passHash):
         self.status = status
         self.email = email
         self.passHash = passHash
+
     def authenticate(self, password):
         return self.passHash == bcrypt.hashpw(password, self.passHash)
+
     def encode_auth_token(self):
```

Comment:
JWT encode and decode methods are really starting to look out of place in `attendee.py`, we should shift them to somewhere else like a `jwt.py` or something. Not sure how Python people handle their utility/helper stuff... that's up to you I guess

Reply:
The encode method can be rewritten to take the id of whatever